libopae/src/mmio.c

```diff
@@ -45,39 +45,40 @@
 #define AFU_SIZE	0x40000
 #define AFU_OFFSET	0
 
-fpga_result __FPGA_API__ fpgaWriteMMIO32(fpga_handle handle,
-					 uint32_t mmio_num,
-					 uint64_t offset,
-					 uint32_t value)
+static fpga_result port_get_region_info(fpga_handle handle,
+				 uint32_t mmio_num,
+				 uint32_t *flags,
+				 uint64_t *size,
+				 uint64_t *offset)
```

Comment:
This diff result is really unfortunate for a review - I simply moved `port_get_region_info()` and `port_mmap_region()` up top because the public API calls now depend on these.